parachain/node/src/command.rs

```diff
@@ -382,3 +451,122 @@ impl CliConfiguration<Self> for RelayChainCli {
 		self.base.base.node_name()
 	}
 }
+
+impl DefaultConfigurationValues for FastChainCli {
+	fn p2p_listen_port() -> u16 {
+		30333
+	}
+
+	fn rpc_listen_port() -> u16 {
+		9944
+	}
+
+	fn prometheus_listen_port() -> u16 {
+		9615
+	}
+}
+
+impl CliConfiguration<Self> for FastChainCli {
```

Comment:
It seems missed:
(1) telemetry configuration;
(2) chain id.
Please verify that all parameters are covert, we could have a call about required parameters, if needed.